task3-select-migrate/homework-selects/sql_questions.sql

```diff
@@ -3,55 +3,491 @@
+where SALARY > (select avg(SALARY) from EMPLOYEES)
+  and DEPARTMENT_NAME like 'IT%';
+
 --35. who earns more than Mr. Ozer.
+select EMPLOYEE_ID,
+       FIRST_NAME || ' ' || LAST_NAME as name
+from EMPLOYEES
+where SALARY > (select SALARY from EMPLOYEES where LAST_NAME = 'Ozer');
+
 --36. which employees have a manager who works for a department based in the US.
+select EMP.EMPLOYEE_ID,
+       EMP.FIRST_NAME || ' ' || EMP.LAST_NAME as name
+from EMPLOYEES EMP
+         inner join EMPLOYEES MGR on EMP.MANAGER_ID = MGR.EMPLOYEE_ID
+         inner join DEPARTMENTS D on MGR.DEPARTMENT_ID = D.DEPARTMENT_ID
+         inner join LOCATIONS L on D.LOCATION_ID = L.LOCATION_ID
+where L.COUNTRY_ID = 'US';
+
 --37. the names of all employees whose salary is greater than 50% of their department’s total salary bill.
+select FIRST_NAME || ' ' || LAST_NAME as name
+from EMPLOYEES E
+where SALARY > (select sum(SALARY) / 2 from EMPLOYEES where E.DEPARTMENT_ID = DEPARTMENT_ID);
+
 --38. the employee id, name ( first name and last name ), salary, department name and city for all
---the employees who gets the salary as the salary earn by the employee which is maximum within the joining person January 1st, 2002 and December 31st, 2003.  
---39. the first and last name, salary, and department ID for all those employees who earn more than the average salary and arrange the list in descending order on salary.
---40. the first and last name, salary, and department ID for those employees who earn more than the maximum salary of a department which ID is 40.
---41. the department name and Id for all departments where they located, that Id is equal to the Id for the location where department number 30 is located.
---42. the first and last name, salary, and department ID for all those employees who work in that department where the employee works who hold the ID 201.
---43. the first and last name, salary, and department ID for those employees whose salary is equal to the salary of the employee who works in that department which ID is 40.
---44. the first and last name, salary, and department ID for those employees who earn more than the minimum salary of a department which ID is 40.
---45. the first and last name, salary, and department ID for those employees who earn less than the minimum salary of a department which ID is 70.
---46. the first and last name, salary, and department ID for those employees who earn less than the average salary, and also work at the department where the employee Laura is working as a first name holder.
+--the employees who gets the salary as the salary earn by the employee which is maximum within the
+-- joining person January 1st, 2002 and December 31st, 2003.
+select EMPLOYEE_ID, FIRST_NAME || ' ' || LAST_NAME as name, SALARY, DEPARTMENT_NAME, CITY
+from EMPLOYEES E
+         left join DEPARTMENTS D on E.DEPARTMENT_ID = D.DEPARTMENT_ID
+         left join LOCATIONS L on D.LOCATION_ID = L.LOCATION_ID
+where SALARY = (select max(SALARY)
```

Comment:
This select only shows the one employee who has the highest salary. Read one more time this task.